docs/flutterweathertutorial.md

```diff
@@ -43,476 +67,2514 @@ We'll be focusing on two endpoints:
 - `/api/location/search/?query=$city` to get a locationId for a given city name
 - `/api/location/$locationId` to get the weather for a given locationId
 
-Open [https://www.metaweather.com/api/location/search/?query=london](https://www.metaweather.com/api/location/search/?query=london) in your browser and you'll see the following response
-
-[london_search.json](_snippets/flutter_weather_tutorial/location_search.json.md ':include')
-
-We can then get the where-on-earth-id (woeid) and use it to hit the location api.
-
-Navigate to [https://www.metaweather.com/api/location/44418](https://www.metaweather.com/api/location/44418) in your browser and you'll see the response for weather in London. It should look something like this:
-
-[london.json](_snippets/flutter_weather_tutorial/location.json.md ':include')
-
-Great, now that we know what our data is going to look like, let’s create the necessary data models.
-
-## Creating Our Weather Data Model
```

Comment:
Can we keep the spirit of these subheadings within the new tutorial? Let's break up the text into more bite-sized steps.